docs/content/usage/api-concepts.md

```diff
@@ -0,0 +1,495 @@
+
+### Structure
+
+```yaml
+apiVersion: kube-bind.io/v1alpha2
+kind: APIServiceExport
+metadata:
+  name: widgets-export-consumer123
+  namespace: kube-bind-provider-ns
+spec:
+  # The CRD being exported
+  resources:
+  - group: example.com
+    resource: widgets
+    versions: ["v1"]
+    
+  # Permission claims for this specific export
+  permissionClaims:
+  - group: ""
+    resource: secrets
+    selector:
+      labelSelector:
+        matchLabels:
+          component: widget-service
+          consumer: consumer123
+          
+  # How isolation is done at the provider side
+  clusterScopedIsolation: Prefixed
+  # Informer scope for this export
+  informerScope: Cluster
+  
+status:
+  conditions:
+  - lastTransitionTime: "2025-11-14T12:02:29Z"
+    status: "True"
+    type: Connected
+  - lastTransitionTime: "2025-11-14T12:02:30Z"
+    status: "True"
+    type: ConsumerInSync
+  - lastTransitionTime: "2025-11-14T12:02:25Z"
+    status: "True"
+    type: ProviderInSync
+```
+
+### Relationship to Templates
+
+```
+APIServiceExportTemplate (1)
+    ↓ (consumer binding creates)
+APIServiceExportRequest (1)
+    ↓ (provider processes)
+APIServiceExport + BoundSchema (1)
+    ↕ (bidirectional sync)
+APIServiceBinding (consumer side)
+```
+
+## APIServiceExportRequest
+
+**Purpose**: Represents a consumer's request to bind to a specific service template.
+
+**Used by**: Service consumers (via CLI/UI)  
+**Scope**: Namespaced (on consumer side)  
+**Lifecycle**: Created during binding process, short lived until APIServiceExport is established.
+
+### Key Features
+
+- **Binding Initiation**: Starts the binding process between consumer and provider
+- **Authentication Context**: Contains OAuth2 flow details and credentials  
+- **Template Reference**: Points to the specific template being requested
+- **Status Tracking**: Reports binding progress and any errors
+
+### Structure
+
+```yaml
+apiVersion: kube-bind.io/v1alpha1
+kind: APIServiceExportRequest  
+metadata:
+  name: my-widget-service-binding
+  namespace: default
+spec:
+  # Reference to the template being bound
+  templateRef:
+    name: widget-service-template
+    
+  # Consumer-specific configuration  
+  resources:
+  - group: example.com
+    resource: widgets
+    versions: ["v1"]
+    
+status:
+  conditions:
+  - lastTransitionTime: "2025-11-14T12:02:25Z"
+    status: "True"
+    type: Ready
+  - lastTransitionTime: "2025-11-14T12:02:25Z"
+    status: "True"
+    type: ExportsReady
+  phase: Succeeded
+```
+
+## APIServiceBinding
+
+**Purpose**: Represents the consumer-side binding to a provider service, containing the applied CRDs and managing the resource synchronization.
+
+**Used by**: Automatically created by consumer konnector agents  
+**Scope**: Namespaced (on consumer side)  
+**Lifecycle**: Created when APIServiceExportRequest is processed, long-lived
+
+### Key Features
+
+- **CRD Management**: Contains the actual CRDs applied to the consumer cluster
```

Comment:
This contradicts the description for BoundSchema, which also claims to "contain the actual CRDs to be bound". I was also surprised that we're stuffing all CRDs into a single Kube object, wouldn't that easily lead to size issues?